instrumentation/instagocb/version.go

```diff
@@ -0,0 +1,6 @@
+// (c) Copyright IBM Corp. 2023
+
+package instagocb
+
+// Version is the instrumentation module semantic version
+const Version = "0.1.0"
```

Comment:
you can leave it as 0.0.0
When you trigger the release GH Action, it will increase the version for you. Otherwise you will get 0.2.0 :)

Reply:
Done.